sql/core/src/test/scala/org/apache/spark/sql/JsonFunctionsSuite.scala

```diff
@@ -709,5 +709,4 @@ class JsonFunctionsSuite extends QueryTest with SharedSparkSession {
           options.asJava)),
       Seq(Row("string")))
   }
-
```

Comment:
Thank you for deletion, but to be complete, you need to recover to the master branch version. You may need something like the following.
```
git checkout -f master sql/core/src/test/scala/org/apache/spark/sql/JsonFunctionsSuite.scala
```